pkgs/by-name/al/alac/package.nix

```
@@ -0,0 +1,42 @@
+{
+  autoreconfHook,
+  fetchFromGitHub,
+  lib,
+  stdenv,
+  testers,
+  unstableGitUpdater,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "alac";
+  version = "0.0.7-unstable-2024-10-16";
+
+  outputs = [
+    "out"
+    "dev"
+  ];
+
+  src = fetchFromGitHub {
+    owner = "mikebrady";
+    repo = "alac";
+    rev = "1832544d27d01335d823d639b176d1cae25ecfd4";
+    hash = "sha256-xRWDeqyJ/BEaSWVlDXgaKIKJuBwM8kJDIATVTVaMn2k=";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+  ];
+
```

```suggestion

  env.NIX_CFLAGS_COMPILE = lib.optionalString stdenv.hostPlatform.isDarwin "-DTARGET_OS_MAC";

```

Tested on aarch64-darwin

---

done